MVVM Example/MVVM Example/Extensions/TableView+Reusable.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Created by Sergey Shulga on 07/04/2017.
+// Copyright (c) 2018 Novoda. All rights reserved.
+//
+
+import Foundation
```

Comment:
this is nowdays so common that we could potentially have a reusable library for this code